File: app/src/main/java/me/kirimin/mitsumine/network/api/BookmarkCountApi.java
package me.kirimin.mitsumine.network.api;

import com.android.volley.RequestQueue;

import rx.Observable;

public class BookmarkCountApi {

    public static Observable<String> request(final RequestQueue requestQueue, final String url) {
        return ApiAccessor.stringRequest(requestQueue, "http://api.b.st-hatena.com/entry.count?url=" + url);
    }
}

File: app/src/main/java/me/kirimin/mitsumine/network/api/BookmarkCountApi.kt
package me.kirimin.mitsumine.network.api

import android.content.Context
import rx.Observable

public class BookmarkCountApi {
    companion object {

        public fun request(context: Context, url: String): Observable<String> {
            return ApiAccessor.stringRequest(context, "http://api.b.st-hatena.com/entry.count?url=" + url)
        }
    }
}
